src/fmu/dataio/_metadata.py

```diff
@@ -50,25 +51,26 @@ def default_meta_dollars() -> dict:
 
 def generate_meta_tracklog() -> list[dict]:
     """Initialize the tracklog with the 'created' event only."""
-
-    dtime = datetime.datetime.now(timezone.utc).isoformat()
-    user = getpass.getuser()
-    sysinfo = {
-        "fmu-dataio": {
-            "version": dataio.__version__,
-        },
-    }
-    _kmd = read_named_envvar("KOMODO_RELEASE")
-    if _kmd is not None:
-        sysinfo["komodo"] = {"version": _kmd}
-
     return [
-        {
-            "datetime": dtime,
-            "user": {"id": user},
-            "event": "created",
-            "sysinfo": sysinfo,
-        }
+        meta.TracklogEvent(
+            datetime=datetime.datetime.now(timezone.utc),
+            event="created",
+            user=meta.User(id=getpass.getuser()),
+            sysinfo=meta.SystemInformation.model_validate(
+                {
+                    "fmu-dataio": meta.VersionInformation(
```

Comment:
I hate this, but i cant get it to work without. Seems like there is something off with the alias thing. Needs more investigation.